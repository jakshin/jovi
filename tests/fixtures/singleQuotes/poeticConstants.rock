(Possessives work in poetic constant literals)
My heart's true
Say my heart

Tommy's nobody
Say Tommy

TOMMY'S mysterious
Say TOMMY

Current Year's 2019
Say Current Year

The string's "somebody else's problem"
Say the string  (Possessives don't get evaluated in assigned string literals)
