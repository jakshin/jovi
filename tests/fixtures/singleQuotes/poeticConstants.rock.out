true
null
mysterious
2019
somebody else's problem
